app/controllers/show-project.js

```diff
@@ -61,6 +61,17 @@ export default class ShowProjectController extends Controller {
     });
   }
 
+  // checks that array of BBLs exists, and if so returns true
+  // this is used to either display the BBLs for a project page on the map (true) OR
+  // to display "No Map Available" when a project has no BBLs (false)
+  @computed('model.bbl_featurecollection')
+  get hasBBLs() {
+    const { bbls } = this.model;
```

Comment:
why can't we just check that model.bbls exists instead of model.bbl_featurecollection in the template?

Reply:
Also I'd add that the dependent key for this computed property is never used here. In theory you could just `bbls` directly on the model but it wouldn't recomputed (I know we'd never reasonably do that but in general). I'd say that `model.bbls` is the dependent key since that's what's being used to determine whether or not it has bbls, no?

Reply:
u right